### PR TITLE
Add a sentence: "preliminary declaration for future change to non-GPL licenses" on pull requests.

### PR DESCRIPTION
**Pull request by dnobori**
Add a sentence: "preliminary declaration for future change to non-GPL licenses" on pull requests.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,3 +12,8 @@ You have two options which are described on the above policy.
 Could you please choose either option 1 or 2, and specify it clearly on the reply?
 
 -
+
+PRELIMINARY DECLARATION FOR FUTURE CHANGE TO NON-GPL LICENSES
```
[the remark this answers is not in the document]
Comment (dnobori) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thank you. I fixed it.

[the remark this answers is not in the document]
Comment (davidebeatrici) [reply .github/PULL_REQUEST_TEMPLATE.md]: You're welcome.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -12,3 +12,8 @@ You have two options which are described on the above policy.
...
+
+I hereby agree in advance that my codes will be licensed automatically under the Apache License or similar BSD/MIT-like open source licenses in the case if the SoftEther VPN Project will adopt to such a non-GPL license in future.
```
[the remark this answers is not in the document]
Comment (dnobori) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thank you. I fixed it, too.

[the remark this answers is not in the document]
Comment (davidebeatrici) [reply .github/PULL_REQUEST_TEMPLATE.md]: You're welcome.

**Comment by moatazelmasry2** [comment]
Nice. But I'd have prefered that this PR be integrated using Squash and merge, instead of merge, because then the whole PR would be added as just one Commit

**Comment by davidebeatrici** [comment]
@dnobori I think that you should use the same email address for Git and GitHub, otherwise the commits you create locally are not counted as contributions in your profile.

**Pull request merged**